.travis.yml

```diff
@@ -1,14 +1,21 @@
 language: python
 python:
-    - "2.7"
-    - "3.4"
+    - 2.7
+    - 3.4
```

Comment:
We want to test on our minimum py3 version, which is 3.4. Since nothing should be broken from 3.4 to 3.5, we assume that it works on 3.5 also.